src/main/java/smokefree/Query.java

```diff
@@ -48,4 +50,19 @@ public Profile profile(DataFetchingEnvironment env) {
     public Playground.PlaygroundObservations validation(String id) {
         return initiatives.validation(id);
     }
+
+    /**
+     * It will compute the total number of volunteers.
+     * @return
+     */
+    // Need to discuss for return format(Int or List).
+    // Whether It should return just int or list of Volunteers(?in this case total count will be calculated at front end, and helps to display all volunteers on click)
```

Comment:
both please. The front-end does two kinds of frontend queries: 
getAllPlaygrounds -> summaries of all playgrounds -> requires only the number of volunteers per playground
getPlaygroundDetails -> for a single playground -> should return all information